dlang.org.ddoc

```diff
@@ -50,30 +52,31 @@ _=
 
 DDLINK=$(LINK2 $1.html, $3)
 DDOC_BACKQUOTED = $(D $0)
-DDOC_SUMMARY   = $0$(P)
-DDOC_DESCRIPTION = $0$(P)
-DDOC_AUTHORS   = $(B Authors:)$(BR)$0$(P)
-DDOC_BUGS      = $(RED BUGS:)$(BR)$0$(P)
-DDOC_COPYRIGHT = $(B Copyright:)$(BR)$0$(P)
-DDOC_DATE      = $(B Date:)$(BR)$0$(P)
-DDOC_DEPRECATED = $(RED Deprecated:)$(BR)$0$(P)
-DDOC_EXAMPLES  = $(B Examples:)$(BR)$0$(P)
-DDOC_HISTORY   = $(B History:)$(BR)$0$(P)
-DDOC_LICENSE   = $(B License:)$(BR)$0$(P)
-DDOC_RETURNS   = $(B Returns:)$(BR)$0$(P)
-DDOC_SEE_ALSO  = $(B See Also:)$(BR)$0$(P)
-DDOC_STANDARDS = $(B Standards:)$(BR)$0$(P)
-DDOC_THROWS    = $(B Throws:)$(BR)$0$(P)
-DDOC_VERSION   = $(B Version:)$(BR)$0$(P)
-DDOC_SECTION_H = $(B $0)$(BR)
-DDOC_SECTION   = $0$(P)
-DDOC_PARAMS    = $(B Parameters:)<table class=parms>$0</table>$(P)
+DDOC_SUMMARY   = $(PC summary, $0)
+DDOC_DESCRIPTION = $(PC description, $0)
+DDOC_AUTHORS   = $(DDOCKEYVAL Authors, $0)
+DDOC_BUGS      = $(DDOCKEYVAL Bugs, $0)
+DDOC_COPYRIGHT = $(DDOCKEYVAL Copyright, $0)
+DDOC_DATE      = $(DDOCKEYVAL Date, $0)
+DDOC_DEPRECATED = $(DDOCKEYVAL Deprecated, $0)
+DDOC_EXAMPLES  = $(DDOCKEYVAL Examples, $0)
+DDOC_HISTORY   = $(DDOCKEYVAL History, $0)
+DDOC_LICENSE   = $(DDOCKEYVAL License, $0)
+DDOC_RETURNS   = $(DDOCKEYVAL Returns, $0)
+DDOC_SEE_ALSO  = $(DDOCKEYVAL SeeAlso, $0)
```

Comment:
This change causes the "See Also:" heading to be squashed into "SeeAlso:" in the HTML output. For example, see: http://dlang.org/phobos/std_string#.translate